tests/tests_pytorch/helpers/runif.py

```diff
@@ -249,6 +251,17 @@ def __new__(
             conditions.append(not _HIVEMIND_AVAILABLE or sys.platform in ("win32", "darwin"))
             reasons.append("Hivemind")
 
+        if min_cuda_gpus_no_init:
```

Comment:
We can replace the logic in the block above with his. No need for different flags.


Also, can you apply this change to the Lite `RunIf`?

Reply:
Was wondering if it'd be okay to change the behavior for all tests, SGTM! I've made both suggested changes in 
a30eca4